bve-derive/src/kvp.rs

```diff
@@ -60,6 +73,18 @@ fn parse_fields(item: &ItemStruct) -> Vec<Field> {
                         _ => unimplemented!(),
                     }
                 }
+                if last_path_str == "HashMap" {
+                    field.kind = FieldKind::Hash;
+                    match &segments.arguments {
+                        PathArguments::AngleBracketed(angled) => {
+                            match angled.args.last().expect("Generic must have args") {
+                                GenericArgument::Type(ty) => field.ty = ty.clone(),
+                                _ => unimplemented!(),
```

Comment:
You're only handling one kind of argument here? Do you have plans to handle others?

Reply:
Nope. If I need to, I will, but I doubt it. This macro only should implement exactly what I need, no more.